spanner/google/cloud/spanner/_helpers.py

```diff
@@ -110,7 +113,7 @@ def _try_to_coerce_bytes(bytestring):
                          'base64-encoded bytes.')
 
 
-# pylint: disable=too-many-return-statements
+# pylint: disable=too-many-return-statements,too-many-branches
```

Comment:
It's comforting that Pylint is at least consistent in what it thinks is "too many" branches/returns!